src/hdmf/common/resources.py

```diff
@@ -424,6 +454,15 @@ def get_key(self, **kwargs):
             else:
                 return self.keys.row[key_idx_matches[0]]
 
+    @docval({'name': 'entity_id', 'type': str, 'doc': 'The ID for the identifier at the resource.'})
+    def get_entity(self, **kwargs):
+        entity_id = kwargs['entity_id']
+        entity = self.entities.which(entity_id=entity_id)
+        if len(entity)>0:
+            return self.entities.row[entity[0]]
+        else:
+            return False
```

Comment:
```suggestion
            return None
```

A `get_x` method typically returns None if the object is not found or raises a ValueError.